packages/web/src/components/drawer/Drawer.module.css

```diff
@@ -17,7 +17,7 @@
 
 .fullDrawer {
   top: 0px;
-  height: 110vh;
+  height: 100vh;
```

Comment:
this is slightly risky, but I've tested a decent amount Will pay attention to QA-ing on stage and call out AJ

Reply:
I'm actually surprised that this was using 110% of the viewport originally.....Why would we even do that 😅